compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -1049,7 +1049,8 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             (args, None)
         };
 
-        let mut copied_constant_arguments = vec![];
+        // Keeps track of temporary allocas whose liftime need to be ended after the call.
+        let mut lifetime_ends_after_call: Vec<(Bx::Value, Size)> = Vec::new();
```

Comment:
Also one thing we kinda lose here with this rename is that most of the entities in this list are arguments or owned by the effective arguments: that's why we accumulate them here through `codegen_argument` calls, then actually handle emitting `llvm.lifetime.end` intrinsics and the like in `do_call`. You have to then go read another hundred lines of code to extract that. We could probably spare a line of commentary here on that subject.